### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by Jamstah**
**Description of the change:**

Fix formatting - bad rendering of the first codeblock and other codeblocks should be nested

**Motivation for the change:**

Hard to read!

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

Skipped changelog - formatting only.

### CODE REVIEW

File: website/content/en/docs/upgrading-sdk-version/v1.36.0.md [hunk 1]
```diff
@@ -5,121 +5,124 @@ weight: 998964000
...
+1) [go/v4] Update your `go.mod` file to upgrade the dependencies and run `go mod tidy` to download them:
+   ```go
+   k8s.io/api v0.29.2
+   k8s.io/apimachinery v0.29.2
+   k8s.io/client-go v0.29.2
+   sigs.k8s.io/controller-runtime v0.17.3
+   ```
+2) [go/v4, helm/v1] The projects are now scaffolded with 0.16.0 version of kube-rbac-proxy. Modify the version of kube-rbac-proxy in the following scaffolded files: - config/default/manager_auth_proxy_patch.yaml     
+   ```diff
+     - gcr.io/kubebuilder/kube-rbac-proxy:v0.15.0
```
Comment (mateusoliveira43) [inline website/content/en/docs/upgrading-sdk-version/v1.36.0.md:18]: I would remove spaces from before `-` or  `+` in diff code blocks, so they render with syntax highlighting

Example without spaces (from https://deploy-preview-6797--operator-sdk.netlify.app/docs/upgrading-sdk-version/v1.36.0/)
Screenshot from 2024-08-03 21-37-48

**Review by mateusoliveira43 — APPROVED** [review]
before https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.36.0/

after https://deploy-preview-6797--operator-sdk.netlify.app/docs/upgrading-sdk-version/v1.36.0/

**Comment by acornett21** [comment]
Hi @Jamstah thanks for the PR, [DCO](https://github.com/operator-framework/operator-sdk/pull/6797/checks?check_run_id=28212538887) is failing, could you please sign your comment, and address PR comments. Thanks.

File: website/content/en/docs/upgrading-sdk-version/v1.36.0.md (hunk 1 above)
Comment (Jamstah) [reply website/content/en/docs/upgrading-sdk-version/v1.36.0.md]: Good call, I'll do that in this pr too.

**Comment by Jamstah** [comment]
DCO fixed.

**Review by acornett21 — APPROVED** [review]

**Pull request merged**